scripts/cloudScript.js

```diff
@@ -4449,19 +4453,19 @@ handlers.fightH = function (args, context) {
         if (response.success) {
             if (response.data.beat>0) {
                 log("Halloween event beat floor "+response.data.beat);
-                award(currentPlayerId,"SG",20);
-                statKong(args.kid,"halloween2019",response.data.beat);
+                award(currentPlayerId,"ZG",20);
```

Comment:
Do we need  to register this new currency to playfab? And if so it's most likely a manual one-time thing, right?

Reply:
Gaia does it (it was his request that we use a new currency code, since playfab doesn't allow him a "set SG to 0 for all" command).